moit-domain/src/main/kotlin/com/mashup/moit/domain/user/UserService.kt

```diff
@@ -0,0 +1,21 @@
+package com.mashup.moit.domain.user
+
+import com.mashup.moit.common.exception.MoitException
+import com.mashup.moit.common.exception.MoitExceptionType
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+@Transactional
```

Comment:
```suggestion
@Transactional(readOnly=true)
```
option 추가해줘야할 것 같습니다!

Reply:
앗 놓쳤네요 반영 했습니다!